src/StorageSync/StorageSync/Az.StorageSync.psd1

```diff
@@ -55,7 +55,8 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.2'; })
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Management.StorageSync.dll', 
-               '.\System.Management.dll', '.\System.CodeDom.dll'
+               '.\System.Management.dll', '.\System.CodeDom.dll', 
+               '.\Microsoft.WindowsAzure.Storage.dll'
```

Comment:
You can't add this dependency.  Storage data plane dependency is taken care of by the Az.Accounts module https://github.com/Azure/azure-powershell/blob/master/src/Accounts/Accounts/Az.Accounts.psd1#L77